packages/gatsby/src/schema/infer/type-conflict-reporter.js

```diff
@@ -22,9 +21,10 @@ const isNodeWithDescription = node =>
 
 const findNodeDescription = obj => {
   if (obj) {
-    const node = findRootNodeAncestor(obj, isNodeWithDescription)
-    if (isNodeWithDescription(node)) {
-      return node.internal.description
+    // TODO: Maybe get this back
```

Comment:
Not sure if this ever happened, not sure how to make it work now. I think it's not a big deal, but I can try to restore that behaviour otherwise.